### ISSUE TITLE
Add a CONTRIBUTING.md file to guide new contributors

### ISSUE THREAD

**Issue by mulu-ken** [DapperLib/Dapper#2088]
As a potential contributor to the Dapper project, I noticed the absence of a '**CONTRIBUTING.md**' file in the repository. This document typically provides guidance to people who are considering contributing to a project and establishes clear expectations for conduct and contribution. 
I would suggest creating a 'CONTRIBUTING.md' file that includes at least the following sections:
-  **How to report a bug**
-  **How to suggest a new feature**
-  **Workflow for submitting pull requests**
-  **Coding conventions and standards**
-  **Link to any related documentation**

Please consider this suggestion as it will improve our collaborative experience in Dapper project

**Comment by Manuel-dev-ing** [comment]
Hi @mulu-ken, Can I take up this issue?

**Comment by mgravell** [comment]
I totally get the point of this if there was something unusual involved, but: there isn't. This is all just standard vanilla GitHub. Would it hurt to add a file? Probably not, but it'll just say "use normal GitHub" multiple times....

**Comment by Manuel-dev-ing** [comment]
I agree with you. The official documentation already covers many aspects; I believe the CONTRIBUTING.md file creates a unique identity for the repository, helps contributors become familiar with the workflow, and serves as a practical guide for inexperienced developers who want to contribute for the first time.
